exe/todo

```diff
@@ -13,7 +13,7 @@ class TodoCLI < Thor
     }
     manager = Todo::Manager.new(config)
     if manager.item_already_added?(item)
-      response = ask "\"#{item}\" is already in the list. Add it again? [y/n]:".to_s
+      response = ask("\"#{item}\" is already in the list. Add it again? [y/n]:".to_s)
```

Comment:
Nit: If you switch to single-quotes here, you won't have to escape them.

Reply:
I realized that, is it preferred? Figured it didn't matter either way